### DISCUSSION TITLE
Prevent update of entries in life cycle hook

### DISCUSSION THREAD

**Discussion by Developer-Anjan** [strapi/strapi#8957] [Legacy Topics]
Hello everyone I am here again with another question.

Is it possible to prevent the update operation in lifecycle hooks ( like **beforeUpdate** ) in the community edition?

If yes then how. Please let me know.

Thank you

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
